FDK17プロジェクト/コード/03.サウンド/CSound.cs

```diff
@@ -705,7 +784,7 @@ public BufferFlags DirectSoundBufferFlags
 		/// <para>全インスタンスリスト。</para>
 		/// <para>～を作成する() で追加され、t解放する() or Dispose() で解放される。</para>
 		/// </summary>
-		public static List<CSound> listインスタンス = new List<CSound>();
+		public static readonly ObservableCollection<CSound> listインスタンス = new ObservableCollection<CSound>();
```

Comment:
The List becomes an ObservableCollection, giving the SoundGroupLevelController everything it needs to monitor the list for addition/replacement of sound objects, update levels selectively based on those changes, and to update the entire set of sounds whenever the sound group levels are changed via configuration or keyboard shortcuts.